tests/console.spec.ts

```diff
@@ -30,8 +33,8 @@ test.describe('console input tests', () => {
     ).toHaveCount(1);
   });
 
-  test('object button is created when creating a table', async () => {
-    const tableName = generateVarName('t');
+  test('object button is created when creating a table', async ({ browserName }) => {
+    const tableName = `browserName${generateVarName('t')}`;
```

Comment:
You're not actually using `browserName` here, you're using the string literal "browserName" and the variable `browserName` is unused. Instead pass the `browserName` into the prefix to `generateVarName`:
```suggestion
    const tableName = generateVarName(`${browserName}_t`);
```